src/server/ssrServer.js

```diff
@@ -19,6 +19,7 @@
 /* eslint-disable global-require */
 
 import path from 'path';
+
```

Comment:
```suggestion
```

Reply:
for node specific packages  the following might be clearer
```
`import path from 'node:path'; 
```